com.opera.Opera.yaml

```diff
@@ -0,0 +1,114 @@
+app-id: com.opera.Opera
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.chromium.Chromium.BaseApp
+base-version: '22.08'
+command: opera
+separate-locales: false
+build-options:
+  no-debuginfo: true
+finish-args:
+  - --device=all
+  - --env=GTK_PATH=/app/lib/gtkmodules
+  - --share=ipc
+  - --share=network
+  - --socket=cups
+  - --socket=pcsc # FIDO2
+  - --socket=pulseaudio
+  - --socket=x11
+  - --socket=wayland
+  - --require-version=1.8.2
+  - --system-talk-name=org.freedesktop.UPower
+  - --talk-name=org.freedesktop.FileManager1
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.kde.kwalletd5
+  - --talk-name=org.gnome.SessionManager
+  - --talk-name=org.gnome.Mutter.IdleMonitor.*
+  - --system-talk-name=org.freedesktop.Avahi
+  - --own-name=org.mpris.MediaPlayer2.chromium.*
+  - --filesystem=host-etc
+  - --filesystem=home/.local/share/applications:create
+  - --filesystem=home/.local/share/icons:create
+  - --filesystem=xdg-run/pipewire-0
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-videos
+  - --filesystem=xdg-desktop
+  - --filesystem=/run/.heim_org.h5l.kcm-socket
+  - --persist=.pki
+modules:
+  - name: dconf
+    buildsystem: meson
+    config-opts:
+      - -Dbash_completion=false
+      - -Dman=false
+      - -Dvapi=false
+    cleanup:
+      - /etc
+      - /include
+      - ca.desrt.dconf.service
+      - dconf.service
+      - dconf-service
+      - '*.pc'
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/dconf/0.40/dconf-0.40.0.tar.xz
+        sha256: cf7f22a4c9200421d8d3325c5c1b8b93a36843650c9f95d6451e20f0bcb24533
+        x-checker-data:
+          type: gnome
+          name: dconf
+          stable-only: true
+      # From: https://src.fedoraproject.org/rpms/dconf
+      - type: patch
+        path: dconf-override.patch
+
+  - name: zypak
+    sources:
+      - type: git
+        url: https://github.com/refi64/zypak
+        tag: v2022.04
+        commit: 55a60d110301e8dd37b631503c3524ab7baaf7aa
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
+
+  - name: opera
+    buildsystem: simple
```

Comment:
Is Opera actually redistributable?

Reply:
https://legal.opera.com/eula/computers/

> (B) reproduce and distribute the Software solely as included in an application repository for a desktop open source operating system distribution PROVIDED THAT in all cases the Software is distributed: (i) without modification; (ii) free of charge to end-users; and (iii) with a copy of this EULA. Distribution for embedded open source operating systems is not permitted. For the avoidance of doubt, the Software must be distributed without modification (including as to the default search engine(s) in the Software settings), both at the time of distribution as well as after the Software is installed.

So long as no modifications are made to it outside of what's needed to package it, seems fine.

Reply:
Thank you Oro, I could only find indirect information, that was also a big part of why I wrote to Opera. They should confirm it, I will share their reply once it arrives to avoid any complications.

Reply:
On the note of shared EULA and with example of Chromium, am I correct to assume that packing in for example codecs (ffmpeg) so Opera can play videos by default or any other extensions is not allowed?

Reply:
There's an ffmpeg extension for encumbered codecs, see https://github.com/flathub/org.freedesktop.Platform.ffmpeg